src/pages/provider/index.tsx

```
@@ -26,131 +23,66 @@ export default function Provider() {
-                {provider.modelCount} model(s) found
-              </p>
-              <div className="card-actions justify-end mt-2">
+      <div className="p-4 flex justify-center">
+        <ul className="list bg-base-200 rounded-box w-xl">
+          {providerList && providerList.length <= 0 && (
+            <li className="list-row flex text-center">
+              <p className="w-full">No provider was found.</p>
+            </li>
+          )}
+          {providerList?.map((provider) => (
+            <li className="list-row flex">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)